File: src/main/java/yaasl/server/model/Feedback.java
package yaasl.server.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import static javax.persistence.GenerationType.AUTO;

@Entity
public class Feedback {

    private Long id;
    private String feedback;
    private String comment;

    public Feedback() {
    }

    @Id
    @GeneratedValue(strategy=AUTO)
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getFeedback() {
        return feedback;
    }

    public void setFeedback(String feedback) {
        this.feedback = feedback;
    }

    public String getComment() {
        return comment;
    }

    public void setComment(String comment) {
        this.comment = comment;
    }

}

File: src/main/kotlin/yaasl/server/model/Feedback.kt
package yaasl.server.model

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType.AUTO
import javax.persistence.Id

@Entity
data class Feedback(@Id @GeneratedValue(strategy = AUTO)
                    var id: Long? = null,
                    var feedback: String? = null,
                    var comment: String? = null)
